addon/components/docs-viewer/x-nav/index.hbs

```diff
@@ -1,32 +1,26 @@
-{{#if this.media.isMobile}}
```

Comment:
@MichalBryxi did this leak somehow as pr title only says about `ember-cli-string-helpers`?